Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -94,6 +94,7 @@ LogManager::LogManager()
   m_log[LogTypes::EXPANSIONINTERFACE] = {"EXI", "Expansion Interface"};
   m_log[LogTypes::FILEMON] = {"FileMon", "File Monitor"};
   m_log[LogTypes::GDB_STUB] = {"GDB_STUB", "GDB Stub"};
+  m_log[LogTypes::GDB_THREAD] = { "GDB_THREAD", "GDB Thread" };
```

Comment:
Extra spaces here

(You can run clang-format to automatically fix all style issues, apart from naming)